lib/topics/state_and_behaviour.rb

```diff
@@ -2,4 +2,28 @@
 
 # exercise state and behaviour
 module StateAndBehaviour
+  # exercise car model
+  class Car
+    attr_reader :year, :color, :model, :current_speed
+    def self.default_car
+      Car.new {}
+    end
+
+    def initialize(args)
```

Comment:
`Car.new` is impossible now, you need always pass `Car.new({})` and this is ugly and not ruby-way.
Try some default params. like `args = {}`

And `args` What that mean? Maybe some `car_attributes = {}`